tests/test_authentication_dummy.py

```diff
@@ -13,8 +16,24 @@ class AuthenticationDummyTestCase(
     'Test Authentication Dummy module'
     module = 'authentication_dummy'
 
+    user = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        with Transaction().start(DB_NAME, 0, autocommit=True):
+            User = Pool().get('res.user')
+            # TODO: Factories
+            (user,) = User.create([{
+                'name': 'Test',
+                'login': 'test',
+                'password': 'dontmind'
+            }])
+            cls.user = [user.id, user.login]
```

Comment:
I fear pool objects are not meant to be kept and reused outside their transactions.

Due to the simplicity of the resources to be created for the test (a user) maybe it would be easier to understand the test suite if it is created in test methods that need it, maybe with the aid of a helper method, and wrap the test method with the good old trytond.tests.test_tryton.with_transaction decorator just like https://github.com/tryton/trytond/blob/39d651cad761277eae18cd0b52701eadf4d63669/trytond/tests/test_user.py#L87